### ISSUE TITLE
Error Class "GetRenderProgressResponse" not found

### ISSUE THREAD

**Issue by hagenholm** [remotion-dev/remotion#3097]
Hello, i have a big problem.

I've installed on laravel and have a problem, try these solutions: 

Problem (1):
Try adding:

namespace RemotionLambdaPhp;

In the GetRenderProgressResponse.php No file

Problem (2):
Try adding to all GetRenderProgressResponse
the  remaining in GetRenderProgressResponse

And the only thing that helped me was to include the file in PHPClient.

```
[2023-10-29 01:24:24] local.ERROR: Class "GetRenderProgressResponse" not found {"exception":"[object] (Error(code: 0): Class \"GetRenderProgressResponse\" not found at /XXXXX/vendor/remotion/lambda/src/PHPClient.php:121)
[stacktrace]
```

In line:

`private function handleLambdaResponseProgress(string $response): GetRenderProgressResponse
    {
        $response = json_decode($response, true);

        if (isset($response->errorMessage)) {
            throw new Exception($response->errorMessage);
        }

        $classResponse = new GetRenderProgressResponse(); <--------------- !!
        $classResponse->bucket = $response['bucket'];
        $classResponse->chunks = $response['chunks'];`

Along with that, you should change the Exceptions.php file to ValidationException.php
since it gives error:

`Class RemotionLambdaPhpValidationException located in ./vendor/remotion/lambda/src/Exceptions.php does not comply with psr-4 autoloading standard. Skipping.`

Thanks and I love Remotion.

**Comment by JonnyBurger** [comment]
Hi @hagenholm!

I am just trying to reproduce it on a Laravel app now. Actually I don't get the error it seems, the line numbers also seem a bit shifted, so I wanted to ask which version of Remotion / Lambda PHP SDK you are using?

We included a fix for some broken stuff in https://github.com/remotion-dev/remotion/pull/2639 which was released around v4.0.11. If you confirm that you are above this version, I will apply your suggestions.
